.ci/aarch64/clean_up_aarch64.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 ARM Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+stale_process_union=( "docker-containerd-shim" "qemu-system-aarch64" )
+stale_docker_mount_point_union=( "/var/lib/docker/containers" "/var/lib/docker/overlay2" )
+stale_docker_dir="/var/lib/docker"
+stale_kata_dir="/var/lib/vc/sbs"
```

Comment:
you may as well nuke the whole of `/var/lib/vc` I think. And, there is also a `/run/vc` dir as well I believe.